src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import BootstrapAlert from 'react-bootstrap/Alert'
+import Button from 'react-bootstrap/Button'
+
+interface Props {
+  /** Defines the color of the alert. Defaults to primary.
+   * @default "primary"
+   **/
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'info' | 'light' | 'dark'
+  /** Defines the title of the alert.*/
+  title?: string
+  /** Defines the message of the alert.*/
+  message?: string
```

Comment:
Do we want to make `message` required?

Reply:
Uhm, my though was to leave everything optional because we don't know yet what all the future implementation would be. Maybe there will be usages with just the title...